apps/pay/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.9"
```

Comment:
Not sure if this docker-compose file will be necessary anymore

Reply:
Will remove this